app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_one :users
```

Comment:
**Changes Required**
- [ ] The `Post` belongs to the users. So the right relationship here would be `belongs_to` rather than `has_one`. 
Let us try and make these changes.